typescript/cli/src/context/context.ts

```diff
@@ -66,12 +67,11 @@ export async function getContext({
 }: ContextSettings): Promise<CommandContext> {
   const registry = getRegistry(registryUri, registryOverrideUri, !disableProxy);
 
-  let signer: ethers.Wallet | undefined = undefined;
+  let signer: Signer | Wallet | undefined = undefined;
```

Comment:
does the zksync ethers wallet not implement the ethers Signer interface? If so, there's no need for the union type here.